data/json/items/containers.json

```diff
@@ -1808,5 +1808,31 @@
         "max_contains_weight": "1 kg"
       }
     ]
+  },
+  {
+    "id": "aquarium_medium",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "glass aquarium" },
+    "description": "A 50 gallon glass aquarium, used for safely storing small animals for use as pets and food.",
+    "weight": "83 kg",
+    "volume": "222 L",
+    "longest_side": "91 cm",
+    "price": 0,
+    "bashing": 1,
+    "material": [ "glass" ],
+    "symbol": ")",
+    "color": "light_cyan",
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "rigid": true,
+        "watertight": true,
+        "open_container": true,
+        "max_contains_volume": "189 L",
+        "max_contains_weight": "300 kg"
+      }
+    ],
+    "qualities": [ [ "CONTAIN", 1 ] ]
```

Comment:
What does the petpack use to contain animals? I thought it had it's own data structure for json that you'd need to mirror?

Reply:
I was looking at that too. It's structured as a tool with a couple of special flags I haven't messed with before:

`    "properties": [ [ "creature_size_capacity", "SMALL" ] ],
    "use_action": [ "CAPTURE_MONSTER_ACT" ],`
I'm betting it can be used to capture any small monster, though I don't think you can spawn it containing a monster yet. No idea how it's defining what a SMALL monster is, I thought we killed that off years ago.